src/components/Footer/Footer.js

```diff
@@ -10,9 +10,22 @@ import MemoIc_block_inactive from '../../assets/block/Ic_block_inactive';
 import MemoIc_comment from '../../assets/icons/Ic_comment';
 import MemoIc_share from '../../assets/icons/Ic_share';
 import Memoic_globe from '../../assets/icons/ic_globe';
+import MemoIc_senddm from '../../assets/icons/ic_send_dm';
+import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
+import SendDMBlack from '../../assets/icons/images/send-dm-black.svg';
+import ShareBlack from '../../assets/icons/images/share-black.svg';
+import {Context} from '../../context';
+import useDMMessage from '../../hooks/core/chat/useDMMessage';
+import useCreateChat from '../../hooks/screen/useCreateChat';
+import {colors} from '../../utils/colors';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
+import dimen from '../../utils/dimen';
+import {normalizeFontSize} from '../../utils/fonts';
 import {FONTS} from '../../utils/theme';
+import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
 
 const Footer = ({
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New imports have been added and the Footer component is now receiving additional props. Ensure that these new props are properly validated using PropTypes to maintain type safety and predictability in your codebase.
```javascript
Footer.propTypes = {
  // existing prop types...
  onPressScore: PropTypes.func,
  isShowDM: PropTypes.bool,
  // add prop types for any other new props...
};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->